src/app/components/comparation-card/comparation-card.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { select, Store } from '@ngrx/store';
+import { tap } from 'rxjs/operators';
+import { Pokemon } from '../../models/pokemon.model';
+import * as PokemonActions from '../../store/actions/pokemon.actions'
+import { selectFromStore } from '../../store/selectors/pokemon.selectors';
+
+@Component({
+  selector: 'app-comparation-card',
+  templateUrl: './comparation-card.component.html',
+  styleUrls: ['./comparation-card.component.css']
+})
+export class ComparationCardComponent implements OnInit {
+  show: boolean;
+  pokemon: Pokemon;
+
+  constructor(private store: Store, private actions$: Actions) {
+    this.store.pipe(
+      select(selectFromStore)
+    ).subscribe(
```

Comment:
every subscription requires a unsubscription or a take/takeUntil pipe to remove it when the component is destroyed to avoid memory leaks